internal/models/secret.go

```diff
@@ -1,4 +1,4 @@
-package secretkey
```

Comment:
secretkey was I think an unnecessary layer of abstraction. Moved it all into secret.